docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,73 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+List all the `tables` from the database with `database-name` and `is temporary table`.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameter
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Listing the tables from this `database`.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     A regex pattern that is used to filter out unwanted functions.
+       <br> - Only `*` and `|` are allowed as wildcard pattern.
+       <br> - Excluding `*` and `|` the remaining pattern follows the regex semantics.
+       <br> - The leading and trailing blanks are trimmed in the input pattern before processing.  
+  </dd>
+</dl>
+
+### Example
+{% highlight sql %}
+-- List all tables in default database
+SHOW TABLES;
++-----------+------------+--------------+--+
```

Comment:
Nit: two space align for results to match the other prs ?

Reply:
Handled